copier/tools.py

```diff
@@ -103,15 +103,29 @@ def to_nice_yaml(data: Any, **kwargs) -> str:
     return result or ""
 
 
+def get_env(
+    envops: EnvOps,
+    filters: Optional[Dict[str, Callable]] = None,
+    paths: Optional[Union[str, Iterable[str]]] = None,
```

Comment:
Should I add `LoaderPaths = Union[str, Iterable[str]]` in `copier.types`?

Reply:
As you please.